quarkus/rebuild.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
```

Comment:
I found this script extremely useful while developing, but happy to remove it if we don't aim to "streamline" the development workflows.

Reply:
+1 to remove it.

Reply:
removed 👍